src/os/src/detail/osTime.c

```diff
@@ -557,15 +557,23 @@ int64_t taosTimeSub(int64_t timeStamp, int64_t duration, char unit, int32_t prec
   return (int64_t)(mktime(&tm) * TSDB_TICK_PER_SECOND(precision));
 }
 
-
 int32_t taosTimeCountInterval(int64_t skey, int64_t ekey, int64_t interval, char unit, int32_t precision) {
```

Comment:
The taosTimeCountInterval function is to count the number of aligned local time intervals to cover the closed range of time `[skey, ekey]`.

Reply:
For example, if the closed time range of [skey, ekey] represented as ASCII time string is ['2020-01-01 23:0:0', '2020-01-02 1:0:0'] and the interval is 1day. Then the number of days to cover the closed time range [skey, ekey] is 2, instead of 1 as computed by the original code.